client/signup/config/steps-pure.js

```diff
@@ -26,7 +26,6 @@ export function generateSteps( {
 	addAddOnsToCart = noop,
 	createAccount = noop,
 	createSite = noop,
-	createWpForTeamsSite = noop,
```

Comment:
This one is still used:

https://github.com/Automattic/wp-calypso/blob/4afcedf36ad7a858f61ac5d087015ca9471af278/client/signup/config/steps.js#L8

Let's remove that usage and `createWpForTeamsSite` altogether.

Reply:
Remove it in e36f4f58785